pymathics/graph/base.py

```diff
@@ -71,7 +72,7 @@
 import networkx as nx
 
 SymbolDirectedEdge = Symbol("DirectedEdge")
```

Comment:
Probably what we should do is create  a `pymathics.craph.graphicssymbols` module to put all symbols in like we do for `systemsymbols`.  In the long long term, I think there may be an image loading mechanism like there is for Lisp and Smalltalk systems, that can dump pickle'd symbol objects (and other objects like this) and then just get these loaded by reading in the PCL file.